### PR TITLE
EP docs

### PR DESCRIPTION
**Pull request by akoumpa**
# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Jenkins CI
To run Jenkins, a NeMo User with write access must comment `jenkins` on the PR.

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Comment by akoumpa** [comment]
jenkins

File: docs/source/nlp/nemo_megatron/parallelisms.rst [hunk 1]
```diff
@@ -32,11 +36,22 @@ Pipeline Parallelism
 
 Sequence Parallelism
 ^^^^^^^^^^^^^^^^^^^^
+Sequence Parallelism (SP) splits input sequence into non-overlapping parts and
+assignes each part to a different GPU for processing.
```
Comment (xrennvidia) [inline docs/source/nlp/nemo_megatron/parallelisms.rst:40]: this is what context parallelism is doing, not sequence parallelism.

File: docs/source/nlp/nemo_megatron/parallelisms.rst [hunk 2]
```diff
@@ -32,11 +36,22 @@ Pipeline Parallelism
...
 
 .. image:: images/sp.gif
     :align: center
     :width: 800px
-    :alt: Sqeuence Parallel
+    :alt: Sequence Parallel
+
+Expert Parallelism
+^^^^^^^^^^^^^^^^^^
+With Expert Paralellim (EP) each expert is assigned to a separate GPU.
```
Comment (xrennvidia) [inline docs/source/nlp/nemo_megatron/parallelisms.rst:49]: I think each GPU can have more than one expert even with the EP.

**Comment by xrennvidia** [comment]
LGTM. Thanks.

**Review by xrennvidia — APPROVED** [review]

**Comment by akoumpa** [comment]
jenkins

**Pull request merged**